File: pyramid.py
def pyramid(x):
    for i in range(1, x + 1):
        row = [str(j + (i - 1) * 2) for j in range(1, i + 1)]
        print(" ".join(row))
x = int(input("Enter the number of rows: "))
pyramid(x)

File: pyramid.js
function pyramid(rows) {
    for (let i = 1; i <= rows; i++) {
      let row = "";
      for (let j = 1; j <= i; j++) {
        row += (2 * (i - 1) + j) + " ";
      }
      console.log(row);
    }
  }
  
  const rows = parseInt(prompt("Enter number of rows: "));
  pyramid(rows);